app/javascript/src/NewService/components/ServiceSourceForm.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import React from 'react'
+
+import type {SourceFormType} from 'NewService/types'
+
+const ServiceSourceForm = (props: SourceFormType) => {
+  const {isServiceDiscoveryUsable, serviceDiscoveryAuthenticateUrl, onHandleFormsVisibility, onStartNamespacesFetch} = props
```

Comment:
There are two handlers:
`onChange` to handle the visibility of the forms, for the whole radio group, and `onClick` just for Service Discovery. I preferred to do it this way because New Service will never need to fire `onStartNamespacesFetch `